src/evict/evict_inline.h

```diff
@@ -9,8 +9,17 @@
 #pragma once
 
 /*
+ * NO_AUTO_FORMAT
+ *
  * __wt_evict_aggressive --
- *     Indicate if the eviction is operating in aggressive mode.
+ *     This function indicates when eviction is unable to make any progress for some amount of time.
+ *     As eviction continues to struggle, this function lets the caller know that eviction has
+ *     become inefficient (or made no progress).
+ *
+ *     This function can be called multiple times and is used in various places to determine if
+ *     eviction strategies need to be more forceful due to ongoing inefficiencies. Additionally, it
+ *     serves as a useful indicator of the health of the eviction process, based on which certain
+ *     choices by the caller are made.
```

Comment:
```suggestion
 *     serves as a useful indicator of the health of the eviction process which callers can use to inform their behavior.
```

Reply:
Updated in [73744c3](https://github.com/wiredtiger/wiredtiger/pull/11096/commits/73744c3a2b3e6a9811054165f507c2971025785e)